gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/tasks/LaunchConfigTask.groovy

```diff
@@ -43,6 +43,10 @@ class LaunchConfigTask extends DefaultTask {
             '-verbose:gc'
     ]
 
+    static final List<String> dnsJvmOpts = [
+            '-Dsun.net.inetaddr.ttl=20'
```

Comment:
Are we certain this is in seconds and not milliseconds?

Reply:
Yeah: https://docs.oracle.com/javase/7/docs/technotes/guides/net/properties.html

Reply:
would be good to clarify this in a comment

Reply:
https://github.com/palantir/sls-packaging/pull/274